### PR TITLE
Multi-line support

### PR DESCRIPTION
**Pull request by chadhietala**

### CODE REVIEW

File: tests/index-test.js [hunk 1]
```diff
@@ -269,6 +269,71 @@ QUnit.module('multi-line', function(hooks) {
...
+      let { builders: b } = env.syntax;
+
+      return {
+        TextNode() {
+          return b.text(`here is a single line string<div data-foo="bar"></div>`);
+        },
+      };
+    });
+
+    assert.equal(code, '<div data-foo="bar"></div>here is a single line string');
```
Comment (chadhietala) [inline tests/index-test.js]: Is this correct?

File: tests/index-test.js [hunk 2]
```diff
@@ -269,6 +269,71 @@ QUnit.module('multi-line', function(hooks) {
...
+      let { builders: b } = env.syntax;
+
+      return {
+        TextNode() {
+          return b.text(`{{foo}}here is a single line string{{bar}}`);
+        },
+      };
+    });
+
+    assert.equal(code, '<div data-foo="bar"></div>here is a single line string');
```
Comment (chadhietala) [inline tests/index-test.js]: Is this correct?

**Comment by rwjblue** [comment]
OK, updated the tests (I had made some really stupid mistakes, sorry 😢 ).  Might still need some tweaking but they are at least not completely bonkers now...

**Comment by chadhietala** [comment]
Dont merge this... it just happens to work because my updated tests are crap.

**Comment by rwjblue** [comment]
@chadhietala - OK, all cleaned up and passing now.  I'm going to land and release, but happy to continue iterating if you spot anything wrong...

**Pull request merged**